src/components/app.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react'
+import Svg from './svg'
```

Comment:
I'm being slightly pedantic here: I think this component could have a more descriptive name. If I came at this project brand new as a new developer, I'd think that this component was some sort of SVG helper component (not actually the whole component which renders the graph specific to this app.) Perhaps `ComparativeD3Graph` (that's a terrible name... but it's a lot more descriptive about what it does.)